src/components/screen/Signup.js

```diff
@@ -68,97 +68,90 @@ type State = {
   statusMsg: string;
 };
 
-class Screen extends Component<Props, State> {
-  static navigationOptions = {
-    title: getString('SIGNUP'),
+function Screen(props: Props, state: State) {
+  let [isRegistering, setIsRegistering] = React.useState(false);
```

Comment:
just using `useState` instead of using `React.useState` would be nicer. because you defined the way of destructuring.